app/controllers/application_controller.rb

```diff
@@ -1,2 +1,10 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name]) # 新規登録時にnameというキーのパラメーターを追加で許可する
+    devise_parameter_sanitizer.permit(:account_update, keys: [:name]) #  更新時にnameというキーのパラメーターを追加で許可する
```

Comment:
```suggestion
    devise_parameter_sanitizer.permit(:account_update, keys: [:name]) #  更新時にnameというキーのパラメーターを追加で許可する
```
sign_upに関しては必要無い気がします。